gravitee-apim-console-webui/package.json

```diff
@@ -21,7 +21,7 @@
     "@fontsource/libre-franklin": "4.4.5",
     "@gravitee/ui-analytics": "7.40.0",
     "@gravitee/ui-components": "3.45.2",
-    "@gravitee/ui-particles-angular": "7.48.0",
+    "@gravitee/ui-particles-angular": "7.48.0-optional-utm-params-in-cta-url-23f2c73",
```

Comment:
Requires this [PR](https://github.com/gravitee-io/gravitee-ui-particles/pull/328) to be merged first